src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -215,7 +211,10 @@ protected void onCreate(Bundle savedInstanceState) {
             mStartUrl = oauth.getCodeRequestUrl();
             mQueryParameters = oauth.getAuthorizationEndpointQueryParameters();
         } catch (UnsupportedEncodingException e) {
-            Log.d(TAG, e.getMessage());
+        	//checked exceptions
```

Comment:
Make sure you get this removed. 
For this case, AuthenticationActivity is called in AuthenticationContext, we need to make sure we return the error back to AuthenticationContext.